### PR TITLE
Keycloak 5522

### PR DESCRIPTION
**Pull request by gnodet**

### CODE REVIEW

**Comment by stianst** [comment]
@mposolda not sure we can handle merging this PR until after 3.4.1. Wdyt?

**Comment by mposolda** [comment]
@gnodet Thanks for the PR and sorry it took so long for review. 

@stianst if I understand correctly, for Keycloak 3.X the Fuse 6.3 is still "MUST have", when Fuse 7 is just "Nice to have". Is it correct? 

If I am looking correctly, changes by @gnodet are backwards compatible and shouldn't break 6.3, but not sure at 100%. In theory, it may be sufficient to just run our testsuite, which is using Fuse 6.3, and ensure that it still works, but some risk of breaking things is still there. So I agree that will be safer to merge rather once master is promoted to 4.X.

BTV. Do we need to support both Fuse 6.3 and Fuse 7 in Keycloak 4.x?

**Comment by stianst** [comment]
Is Fuse 7 released? If we can't this PR reviewed/merged and @pdrozd is OK with it we could add Fuse 7 to Keycloak 3.x and RH-SSO 7.2.

**Comment by pdrozd** [comment]
@stianst  The Fuse 7 is not released and they are far away from that. I would prefer to wait with merging to 4.x

**Comment by chirino** [comment]
So when will this get merged?

**Comment by stianst** [comment]
@chirino sent you an email. Let's continue discussion there.

**Comment by janstey** [comment]
@stianst @chirino any update on this one? Not sure where the email thread went :-)

**Comment by stianst** [comment]
@janstey added you to the email thread

File: distribution/adapters/fuse-adapter-zip/pom.xml [hunk 1]
```diff
@@ -124,6 +124,10 @@
             <groupId>org.keycloak</groupId>
             <artifactId>keycloak-jetty92-adapter</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.keycloak</groupId>
+            <artifactId>keycloak-jetty93-adapter</artifactId>
+        </dependency>
```
Comment (vramik) [inline distribution/adapters/fuse-adapter-zip/pom.xml]: According to KEYCLOAK-6076, there has to be transitive dependencies excluded

**Comment by vramik** [comment]
@hmlnarik I've tested this PR with out current fuse6.3 testsuite and all tests passed

**Comment by hmlnarik** [comment]
Thanks @vramik 
@gnodet Could you please update the dependencies (see #4883) and rebase to current master? Then this PR can be merged and we'll continue working on it.

**Review by vramik — APPROVED** [review]

**Review by hmlnarik — DISMISSED** [review]

**Comment by hmlnarik** [comment]
Thanks @gnodet!

**Comment by hmlnarik** [comment]
Replaced by #5216

**Pull request closed without merging**